src/trace-plugin-loader.js

```diff
@@ -116,16 +116,20 @@ function activate(agent) {
       if (!patchSet) {
         // Load the plugin object
         var plugin = originalModuleLoad(instrumentation.file, module, false);
-        patchSet = [];
+        patchSet = {};
```

Comment:
Oh - I think `[version]` is a mistake. It's not seen anywhere else. Can we guarantee that there's only one version of each module we load? I know there are things like `npm-install-package` but I hope that can count as a pathological case.